todo/src/actions/index.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_TASK = 'ADD_TASK';
+export const TOGGLE_TASK = 'TOGGLE_TASK'
+
+export const addTask = taskObj => {
+  return {
+    type: ADD_TASK,
+    payload: taskObj,
```

Comment:
See, now you are passing in an argument/parameter, so payload becomes useful, cause it becomes Dynamic